docusaurus.config.js

```diff
@@ -6,7 +6,14 @@ module.exports = {
   favicon: 'img/favicon.ico',
   organizationName: 'coblox', // Usually your GitHub org/user name.
   projectName: 'comit-network', // Usually your repo name.
+  plugins: ['@docusaurus/plugin-google-analytics', '@docusaurus/plugin-google-gtag'],
```

Comment:
I might be confused but I think we only need one :)
Imho we only want `@docusaurus/plugin-google-gtag`. The other one was the _old way_ of tracking. I'm not 100% sure, so maybe just use one for now and give it a try?